controllers/apply/awards-for-all/form.js

```diff
@@ -456,279 +465,287 @@ module.exports = function({ locale, data = {} }) {
-                            fields.seniorContactName,
-                            includeAddressAndDob() &&
-                                fields.seniorContactDateOfBirth,
-                            includeAddressAndDob() &&
-                                fields.seniorContactAddress,
-                            includeAddressAndDob() &&
-                                fields.seniorContactAddressHistory,
-                            fields.seniorContactEmail,
-                            fields.seniorContactPhone,
-                            fields.seniorContactCommunicationNeeds
-                        ])
-                    }
-                ]
-            }
-        ]
-    };
+                        cy: ``
+                    }),
+                    fields: compact([
+                        fields.seniorContactRole,
+                        fields.seniorContactName,
+                        includeAddressAndDob() &&
+                            fields.seniorContactDateOfBirth,
+                        includeAddressAndDob() && fields.seniorContactAddress,
+                        includeAddressAndDob() &&
+                            fields.seniorContactAddressHistory,
+                        fields.seniorContactEmail,
+                        fields.seniorContactPhone,
+                        fields.seniorContactCommunicationNeeds
+                    ])
+                }
+            ]
+        };
+    }
 
-    const sectionMainContact = {
-        slug: 'main-contact',
-        title: localise({ en: 'Main contact', cy: '' }),
-        summary: localise({
-            en: `Please provide details for your main contact. This will be the first person we contact if we need to discuss your project.`,
-            cy: ``
-        }),
-        steps: [
-            {
-                title: localise({ en: 'Main contact', cy: '' }),
-                fieldsets: [
-                    {
-                        legend: localise({
-                            en: 'Who is your main contact?',
+    function stepMainContact() {
+        return {
+            title: localise({ en: 'Main contact', cy: '' }),
+            fieldsets: [
+                {
+                    legend: localise({
+                        en: 'Who is your main contact?',
+                        cy: ''
+                    }),
+                    get introduction() {
+                        const seniorFirstName = get(
+                            'seniorContactName.firstName'
+                        )(data);
+                        const seniorSurname = get('seniorContactName.lastName')(
+                            data
+                        );
+                        const seniorName =
+                            seniorFirstName && seniorSurname
+                                ? `, ${seniorFirstName} ${seniorSurname}`
+                                : '';
+
+                        return localise({
+                            en: safeHtml`<p>
```

Comment:
Added `safeHtml` here as we are including user-submitted data here. Fixes a potential self-xss vector.